Cabal/doc/developing-packages.rst

```diff
@@ -1475,7 +1475,7 @@ dependencies. The ``cabal outdated`` command is designed to help with
 that. It will print a list of packages for which there is a new
 version on Hackage that is outside the version bound specified in the
 ``build-depends`` field. The ``outdated`` command can also be
-configured to act on the freeze file (both old- and new-style) and
+configured to act on the freeze file (both old- and v2-style) and
```

Comment:
s/old-/v1-/?

Reply:
Won't do. Someone else can do another `sed`.

Reply:
As in, I'm not sure that `old-` won't match something non valid, and I have no time to check. Sorry. Take or leave.